crates/bevy_geometry/Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+name = "bevy_geometry"
+version = "0.1.0"
+authors = ["Aevyrie Roessler <aevyrie@gmail.com>"]
```

Comment:
```suggestion
authors = [
  "Bevy Contributors <bevyengine@gmail.com>",
  "Aevyrie Roessler <aevyrie@gmail.com>",
]
```

Reply:
Yup, I just realized I did this. Thanks!